docs/reference/src/language/builtins/globals.md

```diff
@@ -3,19 +3,21 @@
 
 ## `Palette`
 
-`Palette` give access to different brushes that can be used to to create custom widgets that matches the colors of
-the selected style e.g. fluent, cupertino, material or qt.
+Use `Palette` to create custom widgets that match the colors of
+the selected style e.g. fluent, cupertino, material, or qt.
 
 ### Properties
 
--   **`background`** (_out_ _brush_): Defines the default background brush.
+-   **`background`** (_out_ _brush_): Defines the default background brush. Use this if none of the more specialised background brushes apply.
 -   **`foreground`** (_out_ _brush_): Defines the foreground brush that is used for content that is displayed on `background` brush.
--   **`control-background`** (_out_ _brush_): Defines the default background brush for widgets.
+-   **`alternate-background`** (_out_ _brush_): Defines an alternate background brush.
```

Comment:
Could you include an example where this is used in current styles, to help users decide whether to go for `background` or `alternate-background`?

Reply:
done